src/genezio.ts

```diff
@@ -168,6 +168,7 @@ Use --backend to deploy only the backend application.`,
             });
             exit(1);
         });
+        process.exit(0);
```

Comment:
Why is this necessary? Isn't the function returning anyway and exiting the program?

Reply:
We have a problem on the build machine. If we launch `genezio deploy` from another process, it seems that the process does not exit and we suspect this is because of the stdin read. We do the `process.exit(0)` as a hack for now to unblock this. We should figure it out why the process does not exit.